cmake/SfizzConfig.cmake

```diff
@@ -1,6 +1,6 @@
-# Do not override the C++ standard if set to more than 14
-if (NOT CMAKE_CXX_STANDARD OR CMAKE_CXX_STANDARD LESS 14)
-    set(CMAKE_CXX_STANDARD 14)
+# Do not override the C++ standard if set to more than 11
+if (NOT CMAKE_CXX_STANDARD OR CMAKE_CXX_STANDARD LESS 11)
+    set(CMAKE_CXX_STANDARD 11)
```

Comment:
If `CMAKE_CXX_STANDARD` is user-defined, it will be in cache anyway and `set()` has no effect, so the condition is not very useful.

Reply:
OK so before I do dumb things, I should override the cache here through `set(CMAKE_CXX_STANDARD 11 CACHE INTEGER "CXX standard version" FORCE)`?

Reply:
I was thinking to keep it as `set(CMAKE_CXX_STANDARD 11)`, no conditional.
this would respect use choice if one is given, or use 11 as default.